dev-support/docker/Dockerfile

```diff
@@ -18,234 +17,80 @@
 # Dockerfile for installing the necessary dependencies for building Hadoop.
 # See BUILDING.txt.
 
-FROM ubuntu:xenial
+FROM centos:7
```

Comment:
@GauthamBanasandra  for ubuntu, openjdk-7 not found installation candidate, the error shows as bellow:
```
.....
ERROR [ 8/27] RUN apt-get -q update     && apt-get -q install -y --no-install-recommends openjdk-7-jdk     && apt-get clean     && rm -rf /var/lib/apt/lists/*
#11 94.95 Get:20 http://archive.ubuntu.com/ubuntu xenial-backports/universe amd64 Packages [12.7 kB]
#11 95.07 Fetched 19.4 MB in 1min 34s (205 kB/s)
#11 95.07 Reading package lists...
#11 95.99 Reading package lists...
#11 96.88 Building dependency tree...
#11 97.02 Reading state information...
#11 97.04 Package openjdk-7-jdk is not available, but is referred to by another package.
#11 97.04 This may mean that the package is missing, has been obsoleted, or
#11 97.04 is only available from another source
#11 97.04
#11 97.04 E: Package 'openjdk-7-jdk' has no installation candidate
```

Reply:
That's because Java 7 isn't there in the Ubuntu Xenial toolchain. Have you tried installing the Java 7 by downloading its tarball? Maybe this'll help - https://linuxconfig.org/oracle-java-jdk-7-on-ubuntu-linux-source-or-rpm-installation

I would advise against changing the base image to Centos 7 if this is the only reason. Here are my suggestions -
1. If you still want to use Centos 7, I would suggest that you create a separate Dockerfile for it (like Dockerfile_centos_7) and leave the original Dockerfile as it is.
2. Or, just install Java 7 from the tarball in Dockerfile.

Reply:
@GauthamBanasandra ok, thanks a lot,I will try to fix this bug, and create a separate Dockerfile for centos later.